csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/Usings.cs

```diff
@@ -1 +1,2 @@
-global using Microsoft.VisualStudio.TestTools.UnitTesting;
\ No newline at end of file
+global using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Deprecation;
```

Comment:
Since the namespace was changed, this should be updated

Reply:
Hi @jshergal due to this obsolete method warning errors Jenkins build is failing

Reply:
> Since the namespace was changed, this should be updated

You didn't update the using here in the global using, you deleted it. That is why the build is failing. It also would have failed on your local machine. Be sure to always build before you commit.

Reply:
Here is the output:

> /var/lib/jenkins/workspace/codemaze-build-pr/csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs(14,24): error CS0103: The name 'StringUtils' does not exist in the current context [/var/lib/jenkins/workspace/codemaze-build-pr/csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTests.csproj]
> /var/lib/jenkins/workspace/codemaze-build-pr/csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs(24,26): error CS0103: The name 'StringUtils' does not exist in the current context [/var/lib/jenkins/workspace/codemaze-build-pr/csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTests.csproj]
> 
> Build FAILED.